### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by illusional**

### CODE REVIEW

**Comment by lgruen** [comment]
Hmm, those links might be hard to fix, as the linter won't have access to private repos, right?

**Comment by illusional** [comment]
@lgruen, I had a look at the config, some small fixes for regular GitHub links and blocked `hail batch` / `cpg-infrastructure-private`.

File: markdown_link_check_config.json [hunk 1]
```diff
@@ -1,10 +1,23 @@
 {
+  "ignorePatterns": [
+    {
+      "pattern": "batch.hail.populationgenomics.org.au"
+    },
+    {
+      "pattern": "github.com/populationgenomics/cpg-infrastructure-private"
```
Comment (lgruen) [inline markdown_link_check_config.json:7]: Oh, excellent, that was what I had been looking for!

**Review by lgruen — APPROVED** [review]
🙇

**Pull request merged**